multigeiger/multigeiger.ino

```diff
@@ -201,59 +202,59 @@ const unsigned long GMC_dead_time = 190;  // Dead Time of the Geiger Counter. Ha
 
 //====================================================================================================================================
 // Variables
-volatile bool          isr_GMC_cap_full       = 0;
-volatile unsigned int  isr_GMC_counts         = 0;
-volatile bool          isr_gotGMCpulse        = 0;
-volatile unsigned long isr_count_timestamp    = millis();
+volatile bool          isr_GMC_cap_full = 0;
+volatile unsigned int  isr_GMC_counts = 0;
+volatile bool          isr_gotGMCpulse = 0;
+volatile unsigned long isr_count_timestamp = millis();
 volatile unsigned long isr_count_time_between = micros();
-volatile unsigned int  isr_GMC_counts_2send   = 0;
-volatile unsigned long isr_count_timestamp_2send= micros();
-
-         unsigned int  GMC_counts             = 0;
-         unsigned int  GMC_counts_2send       = 0;
-         unsigned int  accumulated_GMC_counts = 0;
-         unsigned long count_timestamp        = millis();
-         unsigned long count_timestamp_2send  = millis();
-         unsigned long last_count_timestamp   = millis();
-         unsigned long last_count_timestamp_2send = millis();
-         unsigned long accumulated_time       = 0;
-         unsigned int  last_GMC_counts        = 0;
```

Comment:
so we talk about this change?

IMHO:
- aligning it that way is quite some work and likely, some change (e.g. a new long variable name) might need changes in multiple lines just to get it all aligned again.
- so it is easier just not to align.

Of course, the same problem exists with alignment of end-of-line comments.